part-3/part-3.md

```diff
@@ -152,252 +153,289 @@ const source = from([1, 2, 3]).pipe(
+As we can see, **distinctUntilChanged()** skipped only the first duplicates.
 
 #
 
-# **Объединение потоков**
+# **Combining streams**
 
-Иногда бывает так, что нам необходимо правильно обработать значения сразу с нескольких потоков. Для начала давайте просто их создадим, как это делали раньше. Для этого воспользуемся оператором **timer()**.
+Sometimes we deal with tasks that require us to handle several streams simultaneously. For starters, let's create some streams to work with. For that purpose we will use the  **timer()** function.
 
 ## **timer()**
 
-**timer()** создает поток, который будет возвращать новые значения с таймаутом.
+**timer()** creates a stream that emits a value the time (in milliseconds) we provide has passed, essentially working like `setTimeout`, but for streams.
 
 ```ts
-import { timer } from "rxjs";
+import { timer } from 'rxjs';
 
-// Первое значение вернет сразу, а остальные каждую секунду
+// First value will be emitted immediately, the next one in 1000ms
 
 const sourceFirst = timer(0, 1000);
 
-// Выведет в консоль:
-
-//. 0 сразу
+// Will log:
 
-//. 1 через 1.0 sec
+//. 0 immediately
 
-// 2 через 2.0 sec
+//. 1 in 1.0 second
 
-sourceFirst.subscribe((value) => console.log(value));
+sourceFirst.subscribe(value => console.log(value));
 
-// Первое значение через 0.5 секунды, а остальные каждую секунду
+// First value in 0.5 second, next one in 1 second
 
 const sourceSecond = timer(500, 1000);
 
-// Выведет в консоль:
+// Will log:
 
-//. 0 через 0.5 sec
+//. 0 in 0.5 sec
 
-//. 1 через 1.5 sec
+//. 1 in 1.5 sec
 
-// 2 через 2.5 sec
+sourceSecond.subscribe(value => console.log(value));
+```
+
+## **interval**
+Next, let's learn how to create a stream that emits over time, but repeatedly (like `setInterval`). For that purpose, we have the **interval** function:
+
+```ts
+import { interval } from 'rxjs';
+
+const source$ = interval(1000);
+
+// Will log:
 
-sourceSecond.subscribe((value) => console.log(value));
+//. 0 in 1 second
+
+//. 1 in 2 seconds
+
+//. 2 in 3 seconds
+
+// and so on
+
+source$.subscribe(value => console.log(value))
 ```
 
 ## **combineLatest()**
 
-Теперь давайте объединим значения этих потоков! Для этого воспользуемся оператором **combineLatest()**.
+Now let's combine values of those streams! We will learn to use the **combineLatest()** operator.
 
-**combineLatest()** объединяет переданные потоки и возвращает значение из всех последних значений потоков.
+**combineLatest()** unites the values of all the provided streams, end emits the latest emitted values of each one of them in an `Array`. So, for example, if one stream emits, that value will be combined with all the latest values emitted from other `Observables` in that array, and emitted together.
+**Important!** Notice that **combineLatest** will only start emitting after each source `Observable` has emitted at least once. Previous incomplete values will be ignored 
```

Comment:
> Previous incomplete values will be ignored 

What does this mean?

Reply:
For example, if we have combineLatest([interval(500), interval(1000)]), the first emission by interval(500) will be ignored

Reply:
The *incomplete values* term is what's confusing me.

How about:
```suggestion
**Important!** Notice that **combineLatest** only starts emitting after each source `Observable` has emitted at least one value. Values emitted before this moment are ignored by the resulting `Observable`.
```